allennlp/data/dataset_readers/nlvr2.py

```diff
@@ -52,7 +52,6 @@ def __init__(
         image_loader: ImageLoader,
         image_featurizer: GridEmbedder,
         region_detector: RegionDetector,
-        *,
```

Comment:
Why did you take this away? It makes sure we can later re-order the optional parameters without breaking the API.